20616.6 <c-27#121> Created logger
20616.7 <c-27#121> appl_initialize was called
20616.7 <c-27#121:6> Logger was updated
20616.7 <c-27#121:6> Started intializing cube lib
20616.7 <c-27#121:6> Finished intializing cube lib with worker, starting to work
20619.8 <c-27#121:6> requestCubes => [24] Buffer size: 0
20619.8 <c-27#121:6> Sent requestCubes signal to root
20619.8 <c-27#121:6> Received 1 cubes from root
20619.8 <c-27#121:6> The main loop continues
20619.8 <c-27#121:6> Started solving a cube
21116.1 <c-27#121:6> Cube failed
21116.1 <c-27#121:6> Added failed cube
21116.1 <c-27#121:6> returnFailedCubes => [24] Buffer size: 3
21116.1 <c-27#121:6> Sent 1 failed cubes to root
21116.1 <c-27#121:6> Received receivedFailedCubes signal from root
21116.1 <c-27#121:6> requestCubes => [24] Buffer size: 0
21116.1 <c-27#121:6> Sent requestCubes signal to root
21116.1 <c-27#121:6> Received 1 cubes from root
21116.1 <c-27#121:6> The main loop continues
21116.1 <c-27#121:6> Started solving a cube
21170.8 <c-27#121:6> appl_interrupt was called
21170.8 <c-27#121:6> Interrupting worker
21170.8 <c-27#121:6> appl_interrupt was called
21170.8 <c-27#121:6> appl_withdraw was called
21170.8 <c-27#121:6> STOP (0.20s since last cb)
21170.8 <c-27#121:6> Solving interrupted
21170.8 <c-27#121:6> The main loop continues
21170.8 <c-27#121:6> Leaving the main loop due to an interruption
21170.8 <c-27#121:6> Started withdraw thread
21170.8 <c-27#121:6> Finished withdrawing lib
21170.8 <c-27#121:6> Enter destructor of CubeWorker
21170.8 <c-27#121:6> Time waiting for messages: 0.000
21172.7 <c-27#121:6> Enter destructor of CubeWorkerInterface
21172.7 <c-27#121:6> Enter destructor of CubeRoot
21172.7 <c-27#121:6> Finished withdraw thread
21232 <c-27#121:6> Enter destructor
21232 <c-27#121:6> Time working: 554.117
21232 <c-27#121:6> Time suspended: 0.000
21232 <c-27#121:6> Joined cleanup thread
21232 <c-27#121:6> Exit destructor
21232 <c-27#121:6> Destructing logger
